src/components/main/ShowCheckList.jsx

```diff
@@ -0,0 +1,21 @@
+import React, {Component} from 'react'
+import data from './simpleData'
+
+class ShowCheckList extends Component{
+    state={typeList: null}
+    componentWillMount(){
+        this.setState({
+            typeList: data
+        });
+    }
+    render(){
+        const {typeList} = this.state;
+        const id = +this.props.match.params.id;
+        const getTitle = typeList.filter(elem => elem.id === id);
```

Comment:
use **find** instead of **filter** method and move this logic to componentDidMount (will be changed when backend will be implemented)